test/test_objdump.py

```diff
@@ -0,0 +1,17 @@
+from tinygrad import Device
+from tinygrad.helpers import cpu_objdump
```

Comment:
Should be a `disassemble` method on the Compiler, see CUDA/AMD/METAL/QCOM

Reply:
This is a bit outside the intended scope of this pr. Regardless I decided to just use llvm-objdump as it works on windows and macos.